app/specs/controllers/courses/CoursesShowCtrl.spec.js

```diff
@@ -0,0 +1,87 @@
+// disable max-nested-callbacks linter for this test
+// it is good to have a good separation of contexts inside tests
+
+/* eslint-disable max-nested-callbacks */
+
+'use strict';
+
+define(['paw',
+'angular-mocks',
+'controllers/courses/CoursesShowCtrl'],
+function() {
+  describe('Courses Show Ctrl', function() {
+    beforeEach(module('paw'));
+
+    var expectedCourseId = '93.42';
+    var expectedParams = {courseId: expectedCourseId};
+
+    var expectedCourse = {
```

Comment:
Update all this object to match the one modified with the suggestions. Also, use the apiResonses API to store data there and retrieve it on different tests. 
There are few examples of its usage all along other tests.